library/src/kinectDriverOpenNI.cpp

```diff
@@ -89,29 +89,29 @@ bool KinectDriverOpenNI::initialize(Property &opt)
     this->seatedMode=false;
     this->img_width=opt.check("img_width",Value(320)).asInt();
     this->img_height=opt.check("img_height",Value(240)).asInt();
+    this->depth_width=opt.check("depth_width",Value(320)).asInt();
```

Comment:
the newly introduced options `depth_width`, `depth_height` and `device` must be documented in the interface (I guess in the `kinectWrapper::open` method).